lib/enkf/ensemble_config.cpp

```diff
@@ -191,8 +197,8 @@ void ensemble_config_free(ensemble_config_type * ensemble_config) {
 ert_impl_type ensemble_config_impl_type(const ensemble_config_type *ensemble_config, const char * ecl_kw_name) {
   ert_impl_type impl_type = INVALID;
 
-  if (hash_has_key(ensemble_config->config_nodes , ecl_kw_name)) {
-    enkf_config_node_type * node = (enkf_config_node_type *)hash_get(ensemble_config->config_nodes , ecl_kw_name);
+  if (ensemble_config->config_nodes.count(ecl_kw_name) > 0) {
+    const enkf_config_node_type * node = ensemble_config->config_nodes.at(ecl_kw_name);
```

Comment:
in these situations where you have to check whether an object is present, and in that case do operations on it, it is more efficient to do it as follows
```
const auto node_it = ensemble_config->config_nodes.find(ecl_kw_name);
if(node_it != ensemble_config->config_nodes.end) {
    const enkf_config_node_type * node = node_it.second;
    ...
```